src/apps/Settings/HomeSettings.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Button, DropDown, Field, TextInput } from '@aragon/ui'
+
+import { AppType, EthereumAddressType } from '../../prop-types'
+import AragonStorage from '../../storage/storage-wrapper'
+
+import Option from './Option'
+
+const defaultName = 'Use the default'
```

Comment:
this should be defaultOption instead of defaultName, seems confusing

Reply:
Yes, I agree.